src/main/scala/com/ironcorelabs/UserCreateKeyPair.scala

```diff
@@ -0,0 +1,18 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+
+case class UserCreateKeyPair(
+  userEncryptedMasterKey: PrivateKey,
+  userPublicKey: PublicKey,
+  needsRotation: Boolean
+)
+
+object UserCreateKeyPair {
+  def apply(uckp: jsdk.UserCreateKeyPair): UserCreateKeyPair =
+    UserCreateKeyPair(
+      PrivateKey(uckp.getUserEncryptedMasterKey),
```

Comment:
This isn't a private key. It's an encrypted private key. We should use a type that describes that.

Reply:
After talking more, this `userEncryptedMasterKey` was removed from `UserCreateResult`